keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets.py

```diff
@@ -0,0 +1,144 @@
+            "channels": [48, 96, 192, 384],
+            "depths": [1, 3, 3, 1],
+            "include_rescaling": True,
+            "use_depthwise": False,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "cspdarknets": {
+        "metadata": {
+            "description": (
+                "CSPDarkNet model with [64, 128, 256, 512] channels and "
+                "[1, 3, 3, 1] depths where the batch normalization "
+                "and SiLU activation are applied after the convolution layers."
+            ),
+        },
+        "class_name": "keras_cv.models>CSPDarkNetBackbone",
+        "config": {
+            "channels": [64, 128, 256, 512],
+            "depths": [1, 3, 3, 1],
+            "include_rescaling": True,
+            "use_depthwise": False,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "cspdarknetm": {
+        "metadata": {
+            "description": (
+                "CSPDarkNet model with [96, 192, 384, 768] channels and "
+                "[2, 6, 6, 2] depths where the batch normalization "
+                "and SiLU activation are applied after the convolution layers."
+            ),
+        },
+        "class_name": "keras_cv.models>CSPDarkNetBackbone",
+        "config": {
+            "channels": [96, 192, 384, 768],
+            "depths": [2, 6, 6, 2],
+            "include_rescaling": True,
+            "use_depthwise": False,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "cspdarknetl": {
+        "metadata": {
+            "description": (
+                "CSPDarkNet model with [128, 256, 512, 1024] channels and "
+                "[3, 9, 9, 3] depths where the batch normalization "
+                "and SiLU activation are applied after the convolution layers."
+            ),
+        },
+        "class_name": "keras_cv.models>CSPDarkNetBackbone",
+        "config": {
+            "channels": [128, 256, 512, 1024],
+            "depths": [3, 9, 9, 3],
+            "include_rescaling": True,
+            "use_depthwise": False,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+    "cspdarknetx": {
+        "metadata": {
+            "description": (
+                "CSPDarkNet model with [170, 340, 680, 1360] channels and "
+                "[4, 12, 12, 4] depths where the batch normalization "
+                "and SiLU activation are applied after the convolution layers."
+            ),
+        },
+        "class_name": "keras_cv.models>CSPDarkNetBackbone",
+        "config": {
+            "channels": [170, 340, 680, 1360],
+            "depths": [4, 12, 12, 4],
+            "include_rescaling": True,
+            "use_depthwise": False,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+        },
+    },
+}
+
+backbone_presets_with_weights = {
```

Comment:
Ditto on adding underscores to the preset names

Reply:
Done.